source/plugins/go/src/log_loss_monitoring_test.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+func Test_Process_log(t *testing.T) {
+	init_log_loss_monitoring_globals()
+
+	containerID := "containerID"
+	k8sNamespace := "k8sNamespace"
+	k8sPodName := "k8sPodName"
+	containerName := "containerName"
+	logEntry := "1234567890"
+	logTime := "1970/01/01T01:01:01.00001Z"
+
+	enabled = true
+
+	identifier := k8sNamespace + "_" + k8sPodName + "_" + containerName
+	if len(m_bytes_logged_storage.container_identifiers) != 0 {
+		t.Error("m_bytes_logged_storage didn't start empty")
+	}
+
```

Comment:
are not these doing the same check? looks like duplicate?

Reply:
embarasing bug #3, now fixed